pyrmq/consumer.py

```diff
@@ -77,14 +84,37 @@ def __init__(
             retry_delay=self.retry_delay,
         )
 
+        self.retry_queue_name = f"{self.queue_name}.{self.retry_queue_suffix}"
+
+        queue_args = {
```

Comment:
Move this inside `if self.is_dlk_retry_enabled:`, we don't even need the variable, just assign it to the named argument.